db/views/materialized_patient_summaries_v03.sql

```diff
@@ -0,0 +1,416 @@
+        latest_blood_sugar_1.follow_up_facility_name as latest_blood_sugar_1_follow_up_facility_name,
+        latest_blood_sugar_1.follow_up_date as latest_blood_sugar_1_follow_up_date,
+        latest_blood_sugar_1.follow_up_days as latest_blood_sugar_1_follow_up_days,
+
+        latest_blood_sugar_2.id as latest_blood_sugar_2_id,
+        latest_blood_sugar_2.recorded_at as latest_blood_sugar_2_recorded_at,
+        latest_blood_sugar_2.blood_sugar_type as latest_blood_sugar_2_blood_sugar_type,
+        latest_blood_sugar_2.blood_sugar_value as latest_blood_sugar_2_blood_sugar_value,
+        latest_blood_sugar_2.facility_name as latest_blood_sugar_2_facility_name,
+        latest_blood_sugar_2.facility_type as latest_blood_sugar_2_facility_type,
+        latest_blood_sugar_2.district as latest_blood_sugar_2_district,
+        latest_blood_sugar_2.state as latest_blood_sugar_2_state,
+        latest_blood_sugar_2.follow_up_facility_name as latest_blood_sugar_2_follow_up_facility_name,
+        latest_blood_sugar_2.follow_up_date as latest_blood_sugar_2_follow_up_date,
+        latest_blood_sugar_2.follow_up_days as latest_blood_sugar_2_follow_up_days,
+
+        latest_blood_sugar_3.id as latest_blood_sugar_3_id,
+        latest_blood_sugar_3.recorded_at as latest_blood_sugar_3_recorded_at,
+        latest_blood_sugar_3.blood_sugar_type as latest_blood_sugar_3_blood_sugar_type,
+        latest_blood_sugar_3.blood_sugar_value as latest_blood_sugar_3_blood_sugar_value,
+        latest_blood_sugar_3.facility_name as latest_blood_sugar_3_facility_name,
+        latest_blood_sugar_3.facility_type as latest_blood_sugar_3_facility_type,
+        latest_blood_sugar_3.district as latest_blood_sugar_3_district,
+        latest_blood_sugar_3.state as latest_blood_sugar_3_state,
+        latest_blood_sugar_3.follow_up_facility_name as latest_blood_sugar_3_follow_up_facility_name,
+        latest_blood_sugar_3.follow_up_date as latest_blood_sugar_3_follow_up_date,
+        latest_blood_sugar_3.follow_up_days as latest_blood_sugar_3_follow_up_days
+
+    from ranked_blood_sugars latest_blood_sugar_1
+    left outer join ranked_blood_sugars latest_blood_sugar_2 on latest_blood_sugar_2.patient_id = latest_blood_sugar_1.patient_id and latest_blood_sugar_2.rank = 2
+    left outer join ranked_blood_sugars latest_blood_sugar_3 on latest_blood_sugar_3.patient_id = latest_blood_sugar_1.patient_id and latest_blood_sugar_3.rank = 3
+    where latest_blood_sugar_1.rank = 1
+), next_scheduled_appointment as (
+    select distinct on (patient_id)
+        appointments.*,
+        f.id as appointment_facility_id,
+        f.name as appointment_facility_name,
+        f.facility_type appointment_facility_type,
+        f.district as appointment_district,
+        f.state as appointment_state
+    from appointments
+    left outer join facilities f on f.id = appointments.facility_id
+    where appointments.deleted_at is null and appointments.status = 'scheduled'
```

Comment:
The old `next_scheduled_appointment` and this one are slightly different
Old:
```
  (SELECT DISTINCT ON (patient_id) *
      FROM appointments
      WHERE deleted_at IS NULL
      ORDER BY patient_id, device_created_at DESC) AS next_scheduled_appointment ON next_scheduled_appointment.patient_id = p.id
                                                                     AND next_scheduled_appointment.status = 'scheduled'
```
This would get the latest appointment first and include if it was `scheduled`. We are now getting the "latest scheduled appointment" which can be different. 

We decided to keep the latest appointment in the app overdue list https://github.com/simpledotorg/simple-android/pull/3925. Should do the same on server.